### PR TITLE
RequestsActions: fix merging issues

### PR DESCRIPTION
**Pull request by Pineirin**
* closes: inveniosoftware/invenio-requests#264

### CODE REVIEW

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 1]
```diff
@@ -188,6 +188,17 @@ code {
   border: 0 !important;
 }
 
+.media-inline-block > *{
+  display: inline-block;
+}
+
+.computer-flex-header{
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:195]: why do we need `computer-` (device) specific classes? the idea was that with Media component we will not need them anymore

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/modules/dropdown.overrides [hunk 2]
```diff
@@ -11,4 +11,25 @@
 .ui.floating.dropdown > .menu.fluid-menu-mobile{
   width: calc(100vw - 2 * @defaultPadding);
   left: calc(-100vw/2 + 100%/2 + @defaultPadding);
+}
+
+.ui.dropdown .menu .item {
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/modules/dropdown.overrides:16]: please use less notation

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides (hunk 1 above)
Comment (Pineirin) [reply invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:195]: This is for a Jinja template, I could use the "computer only" class but I would create a lot of code duplication.

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/modules/dropdown.overrides [hunk 3]
```diff
@@ -11,4 +11,25 @@
 .ui.floating.dropdown > .menu.fluid-menu-mobile{
   width: calc(100vw - 2 * @defaultPadding);
   left: calc(-100vw/2 + 100%/2 + @defaultPadding);
-}
\ No newline at end of file
+}
+
+.dropdown .item {
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/modules/dropdown.overrides:16]: please use less notation

**Review by kpsherva — APPROVED** [review]

**Pull request merged**